setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup_reqs = ["setuptools_scm"]
 install_reqs = ["pyserial"]
-test_reqs = ["pytest", "pytest-flake8", "pytest-cov", "black==19.10b"]
+test_reqs = ["pytest", "pytest-flake8", "pytest-cov", "black==19.10b", "pytest-black"]
```

Comment:
Is black with this particular version still needed as a requirement? If not then please remove it.

Reply:
Yep, its still required.